ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/suites/UserManagementLimitsIT.scala

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.testtool.suites
+
+import java.util.UUID
+
+import com.daml.error.definitions.LedgerApiErrors
+import com.daml.ledger.api.testtool.infrastructure.Allocation._
+import com.daml.ledger.api.testtool.infrastructure.Assertions._
+import com.daml.ledger.api.testtool.infrastructure.LedgerTestSuite
+import com.daml.ledger.api.v1.admin.user_management_service.{
+  CreateUserRequest,
+  DeleteUserRequest,
+  GrantUserRightsRequest,
+  ListUserRightsRequest,
+  User,
+  Right => Permission,
+}
+import io.grpc.Status
+
+import scala.collection.immutable.Iterable
+
+final class UserManagementLimitsIT extends LedgerTestSuite {
```

Comment:
Ok.

We were afraid that testing of limiting the number of users could be too much time and thus should be an optional test case. However it's now outdate since we are going to support user pagination from the get-go and won't implement max number of users limit.